common/models/wordbreakers/.gitignore

```diff
@@ -1 +1,2 @@
-build/
\ No newline at end of file
+build/
+**/**.inc.ts
```

Comment:
I think we should list this one .inc file directly rather than using the wildcards

Reply:
Huh.  And here I was, thinking we should make this a top-level pattern instead - do we actually check in any of our `*.inc.ts` files?  I thought the whole purpose of `*.inc.ts` was to indicate we generate them and thus don't need to check 'em in.

Reply:
It's not a bad idea but it's not a policy as such at this point. Given there is only the one .inc.ts file here, it's better to specify it explicitly, and if we start to have them proliferate in the future, then we can consider globalizing it.